cypress/e2e/augury.cy.ts

```diff
@@ -81,4 +83,70 @@ describe("Augury", () => {
     // cy.contains("button.btn-danger", "Delete Creative").click();
     // cy.contains("Creative deleted");
   });
+
+  it("checks inventory availability", () => {
+    cy.visit("/availability");
+
+    // Functions to be implemented
+    cy.getStartDateInput().type("2024-07-01");
+    cy.getEndDateInput().type("2024-07-31");
+    cy.selectSlimSelect("#inventory_id", "Acceptance Test Series");
+    cy.selectSlimSelect("#zone", "House Preroll");
+
+    cy.getAvailabilityResults().should("be.visible");
+    cy.getAvailabilityResults().contains("Calculate");
+    cy.getAvailabilityResults().click();
+
+    // page should container "Forecasted Inventory"
+    cy.contains("Forecasted Inventory");
+  });
+
+  it("generates a campaign report", () => {
+    // Assuming we have a campaign with ID 1
+    cy.visit("/reports");
+    cy.contains("Navigate to different types of reports across Dovetail");
+
+    cy.get("#campaign_id + .ss-main").click(); // Open the dropdown
+    cy.contains("Search");
+
+    // The dropdown is disconnected/supra of the #campaign_id element
+    cy.get(".ss-open-below .ss-search input[type=search]").click({ multiple: true });
+    cy.get(".ss-open-below .ss-search input[type=search]").type("C");
+
+    cy.contains("Campaign Acceptance Test");
+    cy.get(".ss-open-below .ss-search input[type=search]").type("{downArrow}{enter}");
+    cy.get("#campaign_id").closest(".card").find(".btn").click();
+
+    cy.contains("Report Builder");
+  });
+});
+
+declare namespace Cypress {
+  interface Chainable<Subject = any> {
+    getStartDateInput(): Chainable<JQuery<HTMLElement>>;
+    getEndDateInput(): Chainable<JQuery<HTMLElement>>;
+    getInventoryTypeSelect(): Chainable<JQuery<HTMLElement>>;
+    getAvailabilityResults(): Chainable<JQuery<HTMLElement>>;
+    selectSlimSelect(selector: string, value: string): Chainable<JQuery<HTMLElement>>;
+  }
+}
+
+// Custom commands that can be chained
+Cypress.Commands.add("getStartDateInput", () => {
```

Comment:
Neat, I've not used these!

I'm curious - do these become global commands?  Like, would this be available in the `feeder.cy.ts` file?

Reply:
I would assume global, but did not get too far into it.  I'd expect some global accessor methods like these could come in handy with the particular markup that's shared between the apps.